src/main/java/edu/davidengel/advancedjava/StockQuoteApplication.java

```diff
@@ -0,0 +1,78 @@
+package edu.davidengel.advancedjava;
+
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import static java.lang.System.out;
+
+public class StockQuoteApplication {
+
+
+    /**
+     * Main for application service - Prints out stock list data
+     * @param args string symbol, string start date, string end date
+     * @throws ParseException
+     */
+    public static void main(String[] args) throws ParseException {
+
+        String symbol = args[0];
+        Date fromDate = new SimpleDateFormat("MM/dd/yyyy").parse(args[1]);
```

Comment:
you always want to make sure a value exists at an index before accessing that index. If you don't you can easily get null pointer exceptions.